groovy-language-server/src/main/java/com/palantir/ls/groovy/GroovyTextDocumentService.java

```
@@ -112,14 +116,13 @@ public GroovyTextDocumentService(CompilerWrapperProvider provider) {
 
     @Override
     public CompletableFuture<List<? extends SymbolInformation>> documentSymbol(DocumentSymbolParams params) {
-        String uri = params.getTextDocument().getUri();
-        Path filePath = Paths.get(uri);
-        if (!filePath.isAbsolute()) {
-            uri = provider.get().getWorkspaceRoot().resolve(uri).toAbsolutePath().toString();
+        Path absoluteUri = getAbsolutePathRelativeToWorkspace(params.getTextDocument().getUri());
+        if (!absoluteUri.toFile().exists()) {
+            log.warn("Calling documentSymbol on non-existing uri {}", absoluteUri.toString());
```

should we throw an exception here versus continuing?